Source/Core/DolphinWX/TASInputDlg.h

```diff
@@ -33,10 +39,14 @@ class TASInputDlg : public wxDialog
   void CreateWiiLayout(int num);
 
 private:
-  static constexpr int ID_C_STICK = 1001;
-  static constexpr int ID_MAIN_STICK = 1002;
-  static constexpr int ID_CC_L_STICK = 1003;
-  static constexpr int ID_CC_R_STICK = 1004;
+  enum : int
+  {
+    ID_C_STICK = 1001,
+    ID_MAIN_STICK = 1002,
+    ID_CC_L_STICK = 1003,
+    ID_CC_R_STICK = 1004
+  };
+
   int m_eleID = 1005;
```

Comment:
Does ```m_eleID``` have anything to do with the enum above? If so, shouldn't 1005 also have an entry into that enum? Is ```m_eleID``` compliant to the current coding style (or should it be addressed in another PR)?

As I'm not familiar with this, feel free to ignore this comment if I'm wrong.